.github/workflows/release-tag.yaml

```diff
@@ -5,107 +5,43 @@ on:
-          push: true
-          tags: saiya/dsps:${{ steps.get-tag.outputs.tag }}
-          build-args: |
-              DSPS_VERSION_ID=${{ steps.get-tag.outputs.tag }}
-
-  js-client-npm-release:
-    name: Publish JS client
-    runs-on: ubuntu-latest
-    steps:
-      - id: get-version
-        run: |
-          version=$(echo "${{github.ref}}" | sed 's/^refs\/tags\/v//')
-          echo "::set-output name=version::$version"
-      - name: Checkout
-        uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '12.x'
-          registry-url: 'https://registry.npmjs.org'
-          scope: '@dsps'
-      - name: Yarn publish
-        run: |
-          yarn install --frozen-lockfile
-          yarn publish --frozen-lockfile --non-interactive --new-version ${{ steps.get-version.outputs.version }} --no-git-tag-version
-        working-directory: client/js
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-  github-release:
-    name: Create GitHub release
-    runs-on: ubuntu-latest
-    steps:
-      - id: get-tag
-        run: |
-          tag=$(echo "${{github.ref}}" | sed 's/^refs\/tags\/v//')
-          echo "::set-output name=tag::$tag"
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: '1.18.3'
-      - name: Build
-        run: |
-          go install
-          make build
-        working-directory: server
-        env:
-          DSPS_VERSION_ID: ${{ steps.get-tag.outputs.tag }}
-
-      # https://zenn.dev/seita/articles/d1dba77043be8fd50eeb
-      - name: Get the version
-        id: get_version
-        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
-      - name: Get commit summary
-        id: get_commit_summary
-        run: |
-          PREVIOUS_TAG=$(git tag --sort=-creatordate | sed -n 2p)
-          echo "PREVIOUS_TAG: $PREVIOUS_TAG"
-          COMMIT_SUMMARY="$(git log --oneline --pretty=tformat:"%h %s" $PREVIOUS_TAG..${{ github.ref }})"
-          COMMIT_SUMMARY="${COMMIT_SUMMARY//$'\n'/'%0A'}"
-          echo ::set-output name=COMMIT_SUMMARY::$COMMIT_SUMMARY
-
-      - id: create_release
-        uses: actions/create-release@v1
+          version: latest
+          workdir: server
+          args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
-          body: ${{ steps.get_commit_summary.outputs.COMMIT_SUMMARY }}
-          draft: true
-          prerelease: true
-      - uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./server/dsps.zip
-          asset_name: dsps.zip
-          asset_content_type: application/zip
+
+  # js-client-npm-release:
```

Comment:
Since we are using js-client, I think this Action should be also maintained using [npm registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry) etc.

But #3, and this PR doesn't include any breaking change, so I'm planning to take care of js-client afterwards.

Reply:
Is this handled in another ticket?
In addition, I would like to increase the node version.

Reply:
@tshohe sorry for late reply. I created a new ticket for our internal JIRA project.